examples/simpleapp/mc_sc_workflow_example.md

```diff
@@ -130,7 +130,7 @@ Generate 450 blocks to enable Sidechain logic with versions support. Otherwise,
 `./zen-cli -regtest generate 450`
```

Comment:
Shouldn't be 480 now?

Reply:
so far it is still 450. Will become 480 in the md file, as soon as we SDK based sidechain will support version 2

Reply:
Ok but now my question is why in examples/evmapp/src/main/resources/sc_evm_settings.conf is 480